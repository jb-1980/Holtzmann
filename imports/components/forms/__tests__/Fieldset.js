import renderer from "react-test-renderer";
import FieldSet from "../Fieldset";

it("displays an empty fieldset to the user", () => {
  const tree = renderer.create(<FieldSet />);
  expect(tree).toMatchSnapshot();
});

it("displays an empty fieldset, with classes to the user", () => {
  const tree = renderer.create(<FieldSet classes="foo" />);
  expect(tree).toMatchSnapshot();
});

it("displays an empty fieldset, with classes to the user", () => {
  const tree = renderer.create(<FieldSet classes="foo" />);
  expect(tree).toMatchSnapshot();
});