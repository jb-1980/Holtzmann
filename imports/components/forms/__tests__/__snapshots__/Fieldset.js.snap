exports[`test displays an empty fieldset to the user 1`] = `
<fieldset
  className="flush-bottom" />
`;

exports[`test displays an empty fieldset, with classes to the user 1`] = `
<fieldset
  className="flush-bottom foo" />
`;
